spec/send_sms_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'send-sms'
+
+describe SMS do
+
+  describe ".send_sms" do
+
+    it "will send a SMS to the cusomter's phone number" do
+      allow(subject).to receive(:send_sms).and_return("SMS sent")
```

Comment:
Again, this isn't testing anything without an expectation.  I'm guessing it's also sending you a message every time you run your tests?

Getting this test to work properly is probably the most difficult part of this challenge - take a look at figuring out how you might do it if you want to put your dependency injection and mocking skills to the test